OneCommunityGlobal/HighestGoodNetworkApp
# Olga - design project details page

# Description
This is the design of the project details page from Phase II. It was created based off the prototype below:

![Figma prototype](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/107726913/398184e1-213a-4cb6-a540-9fb2be04fc8b)

## How to test:


1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. go to  http://localhost:3000/bmdashboard/12345 
4. check the design against the prototype, check for responsiveness in different screen sizes
